firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedInstallation.java

```diff
@@ -53,6 +53,7 @@
      * locally before registering with FIS servers.
      */
     REGISTER_ERROR,
+    AUTHENTICATION_TOKEN_ERROR,
```

Comment:
How about mentioning the difference between REGISTER_ERROR and AUTHENTICATION_TOKEN_ERROR?